custom_data.go

```diff
@@ -75,6 +76,10 @@ func LoadInflections(r io.Reader) error {
 	defer singularMoot.Unlock()
 
 	for s, p := range m {
+		if strings.Contains(s, " ") || strings.Contains(p, " ") {
```

Comment:
Flect supports parts of ident, so multi-word inflection doesn't make sense and it will cause unexpected results